File: program/find-gcd-of-two-numbers/find-gcd-of-two-numbers.py
def findGCD(x, y):
    while x != y:
        if x > y:
            x = x - y
        elif x < y:
            y = y - x
    return x

x = int(input("n1: "))
y = int(input("n2: "))
print(f'GCD of {x} and {y} : {findGCD(x,y)}')

File: program/find-gcd-of-two-numbers/FindGcdOfTwoNumbers.cs
using System;

class GCD {
  static void Main(string[] args) {
    int x = 12;
    int y = 18;
    int gcd = 1;
    int temp;

    if (x > y) {
      temp = x;
      x = y;
      y = temp;
    }

    for(int i = 1; i < (x+1); i++) {
      if (x%i == 0 && y%i == 0)
        gcd = i;
    }

    Console.WriteLine("GCD of "+ x +" and "+ y +" is: "+ gcd);
  }
}
